### PR TITLE
feat(react-nav-preview): Flagship Nav example

### PR DESCRIPTION
**Pull request by mltejera**
Adds styles for the nav component that will be turned into composed components and re-exported over the course of the next few weeks. Updated content to present a more realistic scenario.

Before:
image

After:
image

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx [hunk 1]
```diff
@@ -58,53 +108,70 @@ export const NavDrawerDefault = (props: Partial<NavDrawerProps>) => {
   return (
     <div className={styles.root}>
       <NavDrawer
-        defaultSelectedValue={'10'}
-        defaultSelectedCategoryValue={'8'}
+        defaultSelectedValue={'2'}
```
Comment (spmonahan) [inline packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx:111]: nit
```suggestion
        defaultSelectedValue="2"
```

File: packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx [hunk 2]
```diff
@@ -58,53 +108,70 @@ export const NavDrawerDefault = (props: Partial<NavDrawerProps>) => {
...
+        defaultSelectedCategoryValue={'1'}
```
Comment (spmonahan) [inline packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx:112]:
```suggestion
        defaultSelectedCategoryValue="1"
```

File: packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx [hunk 3]
```diff
@@ -58,53 +108,70 @@ export const NavDrawerDefault = (props: Partial<NavDrawerProps>) => {
...
-          </NavItem>
-          <NavItem icon={<Folder />} target="_blank" onClick={someClickHandler} value="3">
-            Third
-          </NavItem>
-          <NavCategory value="4">
-            <NavCategoryItem icon={<Folder />}>NavCategoryItem 1</NavCategoryItem>
+        <DrawerBody className={styles.drawerBodyOverrides}>
+          <DrawerHeader className={styles.drawerHeaderOverrides}>
+            <DrawerHeaderNavigation className={styles.drawerHeaderNavOverrides}>
+              <Button appearance={'transparent'} icon={<NavigationFilled />} className={styles.hamburger} />
```
Comment (spmonahan) [inline packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx:121]:
```suggestion
              <Button appearance="transparent" icon={<NavigationFilled />} className={styles.hamburger} />
```

File: packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx [hunk 4]
```diff
@@ -32,24 +48,58 @@ const useStyles = makeStyles({
     gridRowGap: tokens.spacingVerticalXXL,
     gridAutoRows: 'max-content',
   },
-
   field: {
     display: 'grid',
     gridRowGap: tokens.spacingVerticalS,
   },
+
+  headingContent: {
```
Comment (spmonahan) [inline packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawerDefault.stories.tsx:56]: Can you elaborate on these styles some? Are these styles meant to be the default visual treatment of `Nav`? Would we expect users to copy-paste them into their code?

Or put another way, why aren't these baked into the `Nav`?

**Comment by lydsmit** [comment]
Hello world!

File: apps/public-docsite-v9/package.json [hunk 5]
```diff
@@ -37,6 +37,7 @@
     "@fluentui/theme-designer": "*",
     "@fluentui/react-swatch-picker-preview": "*",
     "@fluentui/react-motions-preview": "*",
+    "@fluentui/react-nav-preview": "*",
```
Comment (Hotell) [inline apps/public-docsite-v9/package.json:40]: how this landed here ? 

for adding any stage of v9 package to our official docs we have automated process which needs to be followed

- https://github.com/microsoft/fluentui/wiki/Component-Implementation-Guide#preview-release
- https://github.com/microsoft/fluentui/blob/master/tools/workspace-plugin/src/generators/prepare-initial-release/README.md

**Review by Hotell — DISMISSED** [review]
adding preview/compat phase of packages to official docs needs to be done via our automation flow.

also only packages releases to npm can be part of public documentation. either revert these changes to unblock this PR or please run the generator accordingly.

ty

**Comment by Hotell** [comment]
after running the initial release generator, I'd suggest to do that in separate PR as it's not a good practice to combine adding features with initial release.

tip: also please use github re-request review next time if you submitted changes based on the feedback, otherwise it wont show in peoples github inbox , thus will not be able to look it up. 
image

**Review by spmonahan — APPROVED** [review]

**Pull request merged**